src/App.tsx

```diff
@@ -17,18 +49,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                inputState={inputState}
+                setInputState={setInputState}
+                setToggleFilter={setToggleFilter}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!isLoadedTodos
```

Comment:
it can be checked by todos.length, so 'isLoadedTodos' variable will be not needded anymore